### PR TITLE
[VideoHardware] Refresh rate add mode multi (50/60/24p)/(25/30/24p)

### PR DESCRIPTION
**Pull request by Dima73**
-/proc/stb/video/videomode_24hz is drivers VU 4K

The idea is taken from VTI's image (thanks)

### CODE REVIEW

File: lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py [hunk 1]
```diff
@@ -151,6 +158,13 @@ def readPreferredModes(self):
 			print "[VideoHardware] hotplug on dvi"
 			self.on_hotplug("DVI") # must be DVI
 
+	def check24pChoices(self):
```
Comment (stevenhoving) [inline lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py]: is `is24hzAvailable` not a more descriptive function name?
or couldn't you also add this functionality to the `readAvailableModes` function as it seems to me the more appropriate location for this feature.

File: lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 from Tools.CList import CList
 from Tools.HardwareInfo import HardwareInfo
-from os import path
+import os
```
Comment (stevenhoving) [inline lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py]: you could get away with `from os import path, access`. Is there a particular reason you choose to import the whole `os` module?

File: lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py [hunk 3]
```diff
@@ -158,12 +158,12 @@ def readPreferredModes(self):
 			print "[VideoHardware] hotplug on dvi"
 			self.on_hotplug("DVI") # must be DVI
 
-	def check24pChoices(self):
+	def is24hzAvailable(self):
 		try:
-			self.has24pChoices = os.access("/proc/stb/video/videomode_24hz", os.W_OK) and True or False
+			self.has24pAvailable = os.access("/proc/stb/video/videomode_24hz", os.W_OK) and True or False
```
Comment (stevenhoving) [inline lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py]: Maybe `self.is24hzModeAvailable` is a better variable name.

File: lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py (hunk 2 above)
Comment (Dima73) [inline lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py]: This recommendation Milo

Comment (stevenhoving) [inline lib/python/Plugins/SystemPlugins/Videomode/VideoHardware.py]: @Dima73 thanks...

**Pull request merged**